### DISCUSSION TITLE
%E5%85%B3%E4%BA%8E/author

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#449] [Announcements]
# %E5%85%B3%E4%BA%8E/author

❓我是谁 我是一名学生。目前大二在读，方向是前端开发，平时也会使用Java/Python作为辅助语言。 愿意与你一起努力💪 ❔为什么要建立这个网站 学习的过程中总是会遇到许多问题（坑），如果我能够将这些问题（坑）记录下来，放到自己的个人网站上，就能够在日后复习时减少查找资料的时间。有人可能会问：为什么不用现有的平台（CSDN，掘金，知乎等）呢？因为这些终究不是属于自己的平台，需要注册账号和密码，有可能会给自己带来不必要的麻烦。况且，在这个熙熙攘攘的互联网世界里，拥有自己的一方净土作为心灵港湾，多是一件美事。 ❗网站是怎么运营的

https://etheral.cc/%E5%85%B3%E4%BA%8E/author.html

**Comment by yikolemon** [comment]
卷，疯狂的卷

**Reply by etheral12138** [reply]
哈哈，学长好，以后请多多指教！
